Bottom: d18eb968e1c219221b005c91e16c2b2432f978ad
Top:    e40770cd4843dc3ed3a70f00f1219653804e81ea
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:58:40 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
